Basic-Car-Maintenance/Shared/Settings/Views/ContributorsListView.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ContributorsListView.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Yashraj jadhav on 01/10/23.
+//
+
+import SwiftUI
+
+struct ContributorsListView: View {
+    
+    @StateObject var viewModel: SettingsViewModel
+    
+    var body: some View {
+        List {
+            if (viewModel.contributors?.isEmpty) != nil {
+                ForEach(viewModel.contributors!, id: \.self) { contributor in
```

Comment:
Make `Contributor` conform to `Identifiable` to remove the `id: \.self` in this ForEach